Manifest.in

```diff
@@ -1 +1,3 @@
-recursive-include ingenialink/bin *
\ No newline at end of file
+recursive-include ingenialink/bin *
```

Comment:
Does /bin without FoE longer exist?

Reply:
Should have been deleted on this PR https://github.com/ingeniamc/ingenialink-python/pull/518/files

This folder serves no purpose now